components/PyTorch/Train_PyTorch_model/from_CSV/component.py

```diff
@@ -0,0 +1,114 @@
+from kfp.components import create_component_from_func, InputPath, OutputPath
+
+def train_pytorch_model_from_csv(
```

Comment:
Curious: Is there a way to write test for components? For example, expect the output of a Pytorch component to be the same if given identical input.